README.md

```diff
@@ -100,13 +100,13 @@ average(1, 2, 3);
 * [`unary`](#unary)
 
 </details>
-
-### 📚 Array
+  
+### 📚 Array  
 
 <details>
-<summary>View contents</summary>
-
-* [`all`](#all)
+  <summary>View contents</summary>
+  
+  * [`all`](#all)
```

Comment:
There are unnecessary spaces beng added to multiple lines. Can you somehow fix this? It doesn't make a difference in the rendered Markdown, but it might look weird in other Markdown renderers outside of Github.

Reply:
you mean the indentation?

Reply:
Yeah, I mean the rest of the list does not have indentation at the start. This could display quite weird on certain environments, I suspect, as Markdown is not exactly what you would call a strict language, so different implementations interprert such things slightly differently (e.g. the first term might not be part of the list at all).